src/Api/Vault/Controllers/SecurityTaskController.cs

```diff
@@ -88,11 +88,11 @@ public async Task<ListResponseModel<SecurityTasksResponseModel>> ListForOrganiza
     /// </summary>
     /// <param name="orgId">The organization Id</param>
     [HttpGet("{orgId:guid}/metrics")]
-    public async Task<SecurityTaskMetricsResponseModel> GetTaskMetricsForOrg([FromRoute] Guid orgId)
+    public async Task<SecurityTaskMetricsResponseModel> GetTaskMetricsForOrganization([FromRoute] Guid orgId)
```

Comment:
```suggestion
    public async Task<SecurityTaskMetricsResponseModel> GetTaskMetricsForOrganization([FromRoute] Guid organizationId)
```
⛏️ ... and wherever else it's used.

Reply:
Updated! [8731c1a](https://github.com/bitwarden/server/pull/6164/commits/8731c1a2491586475a5896a9392cd0ccadcd8021)